seatunnel-e2e/seatunnel-connector-v2-e2e/connector-cdc-mongodb-e2e/src/test/resources/mongodbcdc_to_mysql.conf

```diff
@@ -29,6 +29,11 @@ source {
     username = superuser
     password = superpw
     schema = {
+      table = "inventory.products"
+      primaryKey {
+        name = "id"
+        columnNames = ["_id"]
+      }
```

Comment:
Can the old config without table execute normally?

Reply:
Modified to be compatible with older versions of single table synchronization